code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -23,13 +23,16 @@ const defaultOptions: FrameworkOptions = {
   staticDir: undefined,
   addScripts: true,
   addMainFile: true,
+  addPreviewFile: true,
```

Comment:
I think this option is never set to false?

Reply:
Right, this is not used at the moment as we still have to do a bit of work on the side of the storybook nuxt module. The idea is that by default one has a zero-config setup for storybook with nuxt (i.e. no `.storybook` folder) and the module provides the necessary config. It would also enable us to specify the storybook config via the `nuxt.config` file. This is the common pattern with nuxt, see https://nuxt.com/docs/getting-started/configuration#external-configuration-files.

Reply:
So should I wait for that work, before merging this PR?

Reply:
cc @tobiasdiez and @chakAs3

Reply:
From my side we can already merge this PR as is and once we support the "no `.storybook`" mode in the nuxt module, I'll open an new PR setting this toggle to true for nuxt projects. Would this be okay with you?

Reply:
Sounds good!